dashboard/app/views/pd/workshop_survey/submitted.html.haml

```diff
@@ -15,10 +15,9 @@
   - if course == Pd::Workshop::COURSE_CSF
 
     %p
-      To help you get started with CS Fundamentals in your classroom, we've created these
+      To help you get started with CS Fundamentals in your classroom, we've created a
       = succeed "." do
-        %a{:href => 'https://www.amazon.com/ideas/amzn1.account.AFRP6N2PQKQ73HHEN6SJNS5ONHEA', :target => '_blank'} Amazon lists
-      Also feel free to source similar items locally and use these lists as a checklist!
+        %a{:href => 'https://docs.google.com/document/d/19BIaGVliOZCciSDmuezFFUhhCjhrB2oXpVLn0cJAibc/edit#heading=h.d1b2pnqkkc57', :target => '_blank'} list of required materials and supplies
```

Comment:
tiny nit: do we want a period at the end of this string?

Reply:
this confused me at first - the period comes from line 19!

Reply:
oooo I wondered what that line did!